phaseshifts/utils.py

```
@@ -0,0 +1,84 @@
+
+
+class FileUtils(object):
+    """Class for performing phase shift related file operations."""
+
+    @staticmethod
+    def expand_filepath(path):
+        """Backwards-compatible helper mirroring module-level expand_filepath."""
+        return expand_filepath(path)
+
+    @staticmethod
+    def copy_files(files, dst, verbose=False):
```

Ruff (0.14.10)

Missing return type annotation for staticmethod `copy_files`

Add return type annotation: `None`

(ANN205)